src/db-util.h

```diff
@@ -23,15 +37,14 @@ class DbHelper
     template <typename T>
     bool writeProperty(QString propName, const T &val);
 
-    bool fetchId(const QString &uri);
-
 private:
     int m_id;
+    bool m_error;
     QString m_tableName;
 };
 
 template <typename T>
-bool DbHelper::fetchProperty(QString propName, T &ret)
+bool DbUtil::fetchProperty(QString propName, T &ret)
```

Comment:
I think this method can actually be made const, since it doesn't actually modify any state inside DbUtil.
This way, we can make things that depend on it const again.